app/server/controllers/UserController.js

```diff
@@ -663,11 +663,30 @@ UserController.updateRecordsWithMissingFields = function(callback) {
     .exec(callback);
 }
 
+/**
+ * Returns if everyone in the team has checked in
+ * @param  {Function} callback [description]
+ */
+UserController.teamIsCompletelyCheckedIn = function(id, callback) {
```

Comment:
This should be using the `teamSizeAccepted` variable

Reply:
Should this be received as a parameter? Or do I get the value from a query in the function?

Reply:
Inside the controller you can first ask the model for the times, and put all the logic inside the callback (See `UserController.admitUser` for a good example)